src/index.js

```diff
@@ -10,10 +10,13 @@ import uid from './uid';
 
 const noop = () => {};
 
-const [onNextFrame, clearNextFrameAction] = isBrowser &&
-  window.requestAnimationFrame
-  ? [window.requestAnimationFrame, window.cancelAnimationFrame]
-  : [setTimeout, clearTimeout];
+const [onNextFrame, clearNextFrameAction] =
+  isBrowser && window.requestAnimationFrame
+    ? [
+      window.requestAnimationFrame.bind(window),
+      window.cancelAnimationFrame.bind(window),
+    ]
+    : [window.setTimeout.bind(window), window.clearTimeout.bind(window)];
```

Comment:
Library might be used outside of the browser context (SSR, tests), so binding with `window` is not the best idea. Better to just create a proxy functions like:
```js
(fn, ms) => setTimeout(fn, ms)
```

Reply:
Done

Reply:
edit: in fact i removed that part of the change altogether since it's not necessary (no browser context = no ie, no need)